EnglishTranslation-master/SKILL.tsv

```diff
@@ -2126,7 +2126,7 @@ SKILL_20150317_002125	Decreases the added accuracy by half, but increases the cr
 SKILL_20150317_002126	Aggressor: Additional Accuracy	어그레서: 추가명중				
 SKILL_20150317_002127	Increases the additional added accuracy by 2% per level and decreases the additional reduced evasion by 2% per level when [Aggressor] is active.	스킬 [어그레서]가 활성화되며 더해지는 명중률이 레벨당 2%가 추가로 더해지고. 감소하는 회피율이 레벨당 2% 추가로 감소합니다				
 SKILL_20150317_002128	Savagery: Critical Rate	새비지리: 치명타발생				
-SKILL_20150317_002129	Critical rate for [Savagery] is increased by 5% per Attribute level.	스킬 [새비지리]의 치명타 확률이 특성 레벨 당 5% 증가합니다.				
+SKILL_20150317_002129	Critical Rate for [Savagery] is increased by 5% per Attribute level.	스킬 [새비지리]의 치명타 확률이 특성 레벨 당 5% 증가합니다.				
```

Comment:
See above comment about the standardization. Also 'Attribute' should be lowercase.